packages/manhattan/src/index.js

```diff
@@ -1,42 +1,46 @@
-/* eslint-disable react/prop-types */
 import React from 'react'
 import PropTypes from 'prop-types'
-import R from 'ramda'
 import { scaleLinear } from 'd3-scale'
-import { max, min } from 'd3-array'
+import { extent } from 'd3-array'
 
 import {
   HUMAN_CHROMOSOMES,
   HUMAN_AUTOSOMES,
 } from '@broad/utilities/src/constants'
 
+
+function randomColor() {
+  const r = Math.floor(Math.random() * 256)
+  const g = Math.floor(Math.random() * 256)
+  const b = Math.floor(Math.random() * 256)
+  return `rgb(${r},${g},${b})`
+}
+
+
 const ManhattanPlot = ({
   data,
-  title = '',
-  width = 900,
-  height = 500,
-  sexChromosomes = false,
-  showAxisBounds = false,
+  title,
+  width,
+  height,
+  includeSexChromosomes,
```

Comment:
This is entirely subjective/aesthetic, but I think code with boolean variables is easier to understand when the variable names clearly convey true/false (ex. `isFoo`, `hasBar`, etc.)

Reply:
Agreed